src/java/org/apache/cassandra/io/util/FileUtils.java

```diff
@@ -920,4 +922,69 @@ public Object getAttribute(String attribute) throws IOException
             return fileStore.getAttribute(attribute);
         }
     }
+
+
+    /**
+     * Moves the files from a directory to another directory.
+     * <p>Once a file has been copied to the target directory it will be deleted from the source directory.
+     * If a file already exist in the target directory a warning will be logged and the file will not
+     * be deleted.</p>
```

Comment:
Nit: it also moves subdirectories.
```suggestion

    /**
     * Moves the contents of a directory to another directory.
     * <p>Once a file has been copied to the target directory it will be deleted from the source directory.
     * If a file already exists in the target directory a warning will be logged and the file will not
     * be deleted.</p>
```